charts/stable/audiobookshelf/Chart.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: v2
+appVersion: "1.6.69"
+description: Audiobookshelf - Self-hosted audiobook server for managing and playing your audiobooks
+name: audiobookshelf
+version: 1.0.0
+kubeVersion: ">= 1.16.0-0"
+keywords:
+  - audiobookshelf
+  - books
+  - audiobooks
+home: https://github.com/k8s-at-home/charts/tree/master/charts/stable/audiobookshelf
+icon: https://github.com/advplyr/audiobookshelf/blob/master/static/Logo.png
+sources:
+  - https://github.com/advplyr/audiobookshelf
+maintainers:
+  - name: tvories
+    email: tvories@gmail.com
+dependencies:
+  - name: common
+    repository: https://library-charts.k8s-at-home.com
+    version: 4.3.0
+annotations:
+  artifacthub.io/changes: |
+    - initial addition of chart
```

```suggestion
    - kind: added
      description: Initial version
```
We use the "extended" variant of the annotation so that we can keep track of the change type.